### ISSUE TITLE
Pending comments are only visible once you enter the edit screen for an individual bookmark

### ISSUE THREAD

**Issue by jasonkgreen** [ckolderup/postmarks#89]
When someone comments on a bookmark via ActivityPub, that bookmark shows no comments in the admin interface. To see that there is a comment pending review, you have to edit the bookmark.  This makes it very hard to find pending comments if you have more than a few bookmarks.

**Comment by steve-bate** [comment]
Agreed. In my fork, I add "hidden" comments to the Network page with a link to the bookmark editor for the related bookmark that I can use to make the comment visible.
